src/transformers/models/llama/tokenization_llama.py

```diff
@@ -87,8 +89,10 @@ def __init__(
             add_eos_token=add_eos_token,
             sp_model_kwargs=self.sp_model_kwargs,
             clean_up_tokenization_spaces=clean_up_tokenization_spaces,
+            legacy=legacy,
             **kwargs,
         )
+        self.legacy = legacy
```

Comment:
Could we add a `logger.warning_once` here if `legacy` is True to inform the user that legacy behaviour is being used and what that means for the outputs